var/spack/repos/builtin/packages/librdkafka/package.py

```diff
@@ -31,3 +31,4 @@ class Librdkafka(AutotoolsPackage):
 
     depends_on("zstd")
     depends_on("lz4")
+    depends_on("curl")
```

Comment:
Similarly (https://github.com/confluentinc/librdkafka/blob/master/CMakeLists.txt#L43-L67) appears to depend on zlib and openssl.

Reply:
Yeah I realized that more dependencies are probably needed. The .pc file that is installed by version 2.6.0 has this:
```
Requires.private: zlib libcrypto libssl libzstd libcurl liblz4
```
I'll update my PR.